api/crio/crio.go

```diff
@@ -199,6 +199,10 @@ func RootfsCheckpoint(ctx context.Context, ctrDir, dest, ctrID string, specgen *
 
 func CRIORootfsMerge(originalImageRef, newImageRef, rootfsDiffPath, containerStorage string) error {
 	//buildah from original base ubuntu image
+	if _, err := os.Stat("buildah"); err != nil {
```

Comment:
sry this won't work, use os.LookPath https://pkg.go.dev/os/exec, os.Stat doesn't check for executables hehe